packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import RatingItem from './RatingItem'
+
+export default class RatingBar extends Component {
+  static propTypes= {
+    theme: PropTypes.object,
+  }
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const { max, rating, backgroundColor, total, theme } = this.props
```

Comment:
None of these props are in `propTypes` except for `theme`.  Can we add them please?